pytorch/buildspec-1-10.yml

```diff
@@ -46,22 +46,94 @@ context:
       target: deep_learning_container.py
 
 images:
-  # BuildEC2CPUPTTrainPy3DockerImage:
+  BuildEC2CPUPTTrainPy3DockerImage:
+    <<: *TRAINING_REPOSITORY
+    build: &PYTORCH_CPU_TRAINING_PY3 false
+    enable_test_promotion: true
```

Comment:
You can set this value to false if SM Training is the only requirement for testing.